packages/dialtone-vue2/components/emoji_picker/emoji_picker.test.js

```diff
@@ -338,10 +372,10 @@ describe('DtEmojiPicker Tests', () => {
     });
 
     it('Should jump to skin selector from emoji-selector', async () => {
-      const firstFlags = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(10) button');
+      const firstCustoms = wrapper.find('.d-emoji-picker__selector .d-emoji-picker__alignment:nth-child(11) button');
```

Comment:
does nth-last-child work here?

Reply:
yeah definitely

Reply:
It works with `:nth-last-child(1)` updated on: [24858f5](https://github.com/dialpad/dialtone/pull/779/commits/24858f5d8193217e6008c8a96eeb4e15ac82707e)